nvr/nvr.py

```diff
@@ -65,8 +66,15 @@ def start_new_process(self, silent):
         args = os.environ.get('NVR_CMD')
         args = args.split(' ') if args else ['nvim']
 
-        pid = os.fork()
-        if pid == 0:
+        os.environ['NVIM_LISTEN_ADDRESS'] = self.address
+
+        try:
+            pid = subprocess.Popen(args).pid
```

Comment:
Unfortunately this leaves me with a hanging shell. Does this really work on Windows?

Let me read up on `subprocess` and play around with it. Maybe it just needs another argument.

Reply:
As I've set `NVR_CMD` to `nvim-qt`, it worked great that I forgot to check about the situation under terminal. :) Anyway, I did a little investigation, it seems under terminal when subprocess.Popen run nvim background, it'll scrape your terminal or cause it hanging. 

I think this is because nvim is designed as an interactive program. If we pass the argument `--headless` to subprocess.Popen, the problem is solved! But that's not what we want. We need nvim to stay acting interactively when we start an new process for users. 

I did more investigation, and came to the conclusion that maybe the simplest solution is we just  launch nvim and let it wait. I will create a commit based on this.

Reply:
But it's likely that nvr starts a long-running nvim process and it feels wrong to have nvr wait in that case. If the work is done, nvr should exit and nvim should continue running.

Reply:
Yeah, I once thought if there was any way we could just let python exit and nvim continue running.  If possible, that would be great